src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -8,12 +8,15 @@
   Button,
   useDisclosure,
   Input,
+  useToast,
+  VStack,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'VStack' is defined but never used.